### PR TITLE
Aligned the buttons for completed_scene.xml in both orientations

### PR DESCRIPTION
**Pull request by shamathmika**
@chhavip 
Made the changes for issue #119 . Please do review and let me know if there's any more changes to be made. :)

### CODE REVIEW

**Comment by chhavip** [comment]
Will do.

File: PowerUp/app/src/main/res/layout/completed_scene.xml [hunk 1]
```diff
@@ -8,24 +8,23 @@
...
-        android:textSize="@dimen/game_text_size" />
+        android:textSize="@dimen/game_text_size"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true" />
 
     <Button
         android:id="@+id/ContinueButtonMap"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginBottom="69dp"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/completed_scene.xml]: Do not hardcode values, create a dimens entry

File: PowerUp/app/src/main/res/layout/completed_scene.xml [hunk 2]
```diff
@@ -8,24 +8,23 @@
         android:id="@+id/completed_scene_textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentTop="true"
-        android:layout_marginRight="@dimen/completed_scene_margin_right"
-        android:layout_marginTop="@dimen/completed_scene_margin_top"
+        android:layout_marginTop="107dp"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/completed_scene.xml]: Do not hardcode values

File: PowerUp/app/src/main/res/layout-land/completed_scene.xml [hunk 3]
```diff
@@ -20,23 +20,19 @@
         android:id="@+id/ContinueButtonMap"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/completed_scene_continue_button_map_land_margin_bottom"
-        android:layout_marginRight="@dimen/completed_scene_continue_button_map_land_margin_right"
+        android:layout_marginBottom="50dp"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout-land/completed_scene.xml]: Remove hardcoded value

**Comment by shamathmika** [comment]
@chhavip https://github.com/shamathmika/powerup-android/commit/aa6637615ba7e066ff836648a4e98c3ee40049df 
Is this okay?

**Comment by chhavip** [comment]
@shamathmika yes perfect, but could you squash the commits into one as one feature should be in one commit. Please refer https://github.com/chhavip/Git-Guide  or ask if you run into difficulties for the same.

**Comment by aanchal07** [comment]
Changes pending

**Comment by shamathmika** [comment]
@chhavip hi,
I have a small doubt regarding the squash. I don't have the laptop I made these changes on, currently. Can I squash anyway? Do I download it and then combine them? How exactly do I do it?

**Pull request closed without merging**